contrib/upterm/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "upterm"
+pkgver = "0.14.3"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["./cmd/upterm"]
+make_install_args = make_build_args
+hostmakedepends = ["go"]
+pkgdesc = "Instant terminal sharing"
+maintainer = "ttyyls <contact@behri.org>"
+license = "Apache-2.0"
+url = "https://github.com/owenthereal/upterm"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "3ca2bb64943ba8ecd31ae83036794749bb32618a1929f4b22d535cfb28dda367"
+# cross: generates completions with host binary
+# check: uses -race that conflicts with -buildmode=pie
```

Comment:
then just don't pass -race

Reply:
most of them fail without it

Reply:
i just ran them and they work fine, you just forgot bash in checkdepends

Reply:
and when they fail (by hanging) the failure has nothing to do with -race